.github/workflows/pythonapp.yml

```diff
@@ -288,7 +288,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip setuptools wheel
-          pip install -r requirements.txt -r doc/requirements.txt
+          pip install . -r doc/requirements.txt
       - name: Test to check for CVEs for python requirements and HTML report dependencies
```

Comment:
As far as I understand it, we were not installing the current version of cve-bin-tool to be used (hence the . to specify the updated version)

Reply:
> we were not installing the current version of cve-bin-tool to be used

I didn't get this part. Did you mean current versions of dependencies?

Reply:
No, I mean the cve-bin-tool as a whole (instead of pypi we get it from setup.py i.e. the current copy on which the tests are running)

Reply:
No, I mean the cve-bin-tool as a whole (instead of pypi we get it from setup.py i.e. the current copy on which the tests are running)

Reply:
The old one just installs all required packages for `cve-bin-tool` but not the tool itself. The new one install the tool (with all dependencies of course).

This is needed to fix the test, copying my comments from gitter
> this is an interesting one, I'm actually not sure how this test worked before because it doesn't install cve-bin-tool explicitly
in .github/workflows/pythonapp.yml replace line 291 with this one
```diff
-pip install -r requirements.txt -r doc/requirements.txt
+pip install . -r doc/requirements.txt
```
> it's because of the caching that we use. This job workflow has the same cache key as other jobs so when restoring the cache it already had cve-bin-tool installed
in other jobs it is explicitly reinstalled to use the latest commit and in this job we ended up with some old version and never reinstalled the tool